tools/driver/cling.cpp

```diff
@@ -106,6 +107,29 @@ int main( int argc, char **argv ) {
     Interp.loadFile(Lib);
 
   cling::UserInterface Ui(Interp);
+
+  // Get the home directory
+  std::string HomeDir;
+#ifdef __unix__
+  HomeDir = getenv("HOME");
+  HomeDir += "/";
+#elif defined(_WIN32)
+  HomeDir = getenv("USERPROFILE");
+  HomeDir += "\\";
+#endif
+
+  // Look for .clinginit in the home directory and if found process it like a cling script
+  std::string CLingInit = HomeDir + ".clinginit";
+  const std::string Filepath = Interp.lookupFileOrLibrary(CLingInit);
```

Comment:
What is this adding? How is `Filepath` different from `CLingInit`? If this is about "does the file exist", you might as well just create the `ifstream` and check how it's doing.